.github/workflows/IR.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: IR tests
+
+defaults:
+  run:
+    # To load bashrc
+    shell: bash -ieo pipefail {0}
+
+on:
+  pull_request:
+    branches: [master, dev]
+  schedule:
+    # run CI every day even if no PRs/merges occur
+    - cron: '0 12 * * *'
+
+jobs:
+  build:
+    name: IR tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-2022]
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.6
+
+      - name: Install dependencies
+        run: |
+          # FIXME: patched crytic-compile is used to support Windows.
+          # Install from pip once branch is merged and crytic-compile released
+          pip install "crytic-compile@https://github.com/crytic/crytic-compile/archive/refs/heads/dev-windows-long-paths.zip"
+
+          python setup.py install
+          pip install deepdiff
+          pip install pytest
+
+          pip install "solc-select>=v1.0.0b1"
+          solc-select install all
+          solc-select use 0.8.11
+
+          cd tests/test_node_modules/
+          npm install hardhat
+          cd ../..
```

Comment:
From a quick look at the PR it seems you are not using this project, these commands can probably be removed then.

```suggestion
```